lib/errorHandlers/index.ts

```diff
@@ -51,12 +50,10 @@ export function debugError(error: unknown, context?: ApiErrorContext): void {
     logger.debug(i18n(`${i18nKey}.errorOccurred`, { error: String(error) }));
   }
 
-  if (error instanceof Error && error.cause) {
+  if (error instanceof Error && error.cause && !isHubSpotHttpError(error)) {
```

Comment:
We don't want to log a `HubSpotHttpError` because it will contain the full axios error as the cause and will flood the terminal with irrelevant information.